tests/integration/targets/digital_ocean_tag_info/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- block:
+
+    - name: Ensure API key is provided
+      ansible.builtin.fail:
+        msg: do_api_key should be defined in tests/integration/integration_config.yml
+      when:
+        - do_api_key is not defined
+        - do_api_key | length == 0
+
+    - name: Gather tag information
+      community.digitalocean.digital_ocean_tag_info:
+        oauth_token: "{{ do_api_key }}"
+      register: result
+
+    - name: Verify droplet info fetched
```

Comment:
Change `droplet` to `tag` and perhaps `info` to `information` (for consistency).